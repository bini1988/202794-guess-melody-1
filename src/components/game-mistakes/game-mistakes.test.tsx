import React from "react";
import renderer from "react-test-renderer";
import serializer from "jest-emotion";
import GameMistakes from "./game-mistakes";

expect.addSnapshotSerializer(serializer);

it(`GameMistakes correctly renders default markup`, () => {
  const tree = renderer
    .create(
        <GameMistakes mistakes={2}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
