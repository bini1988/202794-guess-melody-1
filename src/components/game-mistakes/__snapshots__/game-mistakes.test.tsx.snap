// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameMistakes correctly renders default markup 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  width: 225px;
}

.emotion-0 svg {
  margin: 0 2px;
}

<div
  className="emotion-0 emotion-1"
>
  <svg
    fill="#cc2c2c"
    height="49"
    width="35"
  >
    <use
      xlinkHref="#note"
      xmlnsXlink="http://www.w3.org/1999/xlink"
    />
  </svg>
  <svg
    fill="#cc2c2c"
    height="49"
    width="35"
  >
    <use
      xlinkHref="#note"
      xmlnsXlink="http://www.w3.org/1999/xlink"
    />
  </svg>
</div>
`;
